rebasehelper/cli.py

```diff
@@ -143,6 +150,13 @@ def add_args(self):
             help="Specify number of retries in case build fails.",
             type=int
         )
+        self.parser.add_argument(
+            "--enable-option",
```

Comment:
Ok. I agree with the --builder at the beginning but I would rather use `--builder-option` cause it logically connects with the builder option rather than with params or args ... How do you see it?